src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -537,7 +547,7 @@ ContainsDisallowedFunctionCallsWalker(Node *expression, WalkerState *state)
 	 * Once you've added them to this check, make sure you also evaluate them in the
 	 * executor!
 	 */
-	StaticAssertStmt(PG_VERSION_NUM <= 90503, "When porting to a newer PG this section"
+	StaticAssertStmt(PG_VERSION_NUM <= 90599, "When porting to a newer PG this section"
```

Comment:
On 2016-07-11 09:43:06 -0700, Brian Cloutier wrote:
> > @@ -537,7 +547,7 @@ ContainsDisallowedFunctionCallsWalker(Node *expression, WalkerState *state)
> >  	 * Once you've added them to this check, make sure you also evaluate them in the
> >  	 * executor!
> >  	 */
> > -	StaticAssertStmt(PG_VERSION_NUM <= 90503, "When porting to a newer PG this section"
> > +	StaticAssertStmt(PG_VERSION_NUM <= 90599, "When porting to a newer PG this section"
> 
> Will the first version of 9.6 be 90601?

No, but does it matter? The relevant part is using a <, instead of <=, no?